Source/i_winmusic.c

```diff
@@ -437,7 +440,34 @@ void I_WIN_PlaySong(boolean looping)
     }
 }
 
-void I_WIN_RegisterSong(void *data, int size)
+static void I_WIN_PauseSong(void *handle)
+{
+    MMRESULT mmr;
+
+    mmr = midiStreamPause(hMidiStream);
+    if (mmr != MMSYSERR_NOERROR)
+    {
+        MidiErrorMessageBox(mmr);
+    }
+}
+
+static void I_WIN_ResumeSong(void *handle)
+{
+    MMRESULT mmr;
+
+    mmr = midiStreamRestart(hMidiStream);
+    if (mmr != MMSYSERR_NOERROR)
+    {
+        MidiErrorMessageBox(mmr);
+    }
+}
+
+static boolean IsMid(byte *mem, int len)
```

Comment:
I think I have seen this function the fourth time now. Maybe we should move it into `i_sound.h`?

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/403/commits/bccf5e796d7786eb60ae8c26231eef697a30fdc7